.vuestorefrontcloud/docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12 AS build
+
+WORKDIR /var/www
+
+COPY . .
+
+RUN cd packages/vue \
+   && yarn \
+   && yarn docs:build
```

Comment:
I noticed there are no images. The problem is that command `yarn docs:build` does not copy the static assets (images) from public folder. If you use comman `yarn build-storybook` instead it should also copy static assets ;)